tests/tensor_search/test_boost_field_scores.py

```diff
@@ -0,0 +1,95 @@
+from marqo.errors import IndexNotFoundError, InvalidArgError
+from marqo.tensor_search import tensor_search
+
+from tests.marqo_test import MarqoTestCase
+
+
+class TestBoostFieldScores(MarqoTestCase):
```

Comment:
Can we test with more documents (10 or so) with varying fields, with varying other params (like pagination)?